wpeverest/everest-forms
# Tweak - Compatibility with newly introduced ThemeGrill Agency Plan.

### All Submissions:






### Changes proposed in this Pull Request:
A new ThemeGrill Agency Plan has been created which has yearly and lifetime subscriptions. Since a new plan has been added, the license of this plan must work with Everest forms. This PR provides compatibility with newy introduced ThemeGrill Agency Plan.

### How to test the changes in this Pull Request:

1. Activate Everest Forms Pro with the license of ThemeGrill Agency plan.
2. Try to install addons from Addons Page, Form Templates while creating new Form and From the addon requiring fields like Captcha, Repeater, Authorize net, Likert, etc. ( PS. Refresh Addons and Templates while doing that )
3. Check if you encounter any error while doing this.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Compatibility with newly introduced ThemeGrill Agency Plan.
